dm/syncer/syncer.go

```diff
@@ -402,6 +405,11 @@ func (s *Syncer) Init(ctx context.Context) (err error) {
 			}
 		}
 	}
+	s.flushCpWorker = &checkpointFlushWorker{
+		input:        make(chan *flushCpTask, 16),
```

Comment:
why chansize is 16

Reply:
It's set by intuition. We just need to ensure that event the checkpoint flush is too slow, once there are too many flush jobs, it will back pressure the sync phase to avoid OOM.